skbio/stats/composition.py

```diff
@@ -634,7 +752,9 @@ def ilr_inv(mat, basis=None, check=True):
     basis : ndarray or sparse matrix, optional
         Orthonormal basis for Aitchison simplex. Defaults to J. J. Egozcue
         orthonormal basis.
-    check : bool
+    axis: int, default -1
+        ignored for backward compactibility
```

Comment:
I'm not crazy about introducing the axis into the function signature at this moment in time. It will make sense once we have tensor support, but for now , it will just confuse users.

I propose to have axis=-1 hard-coded within the function, so that it is not exposed to the user interface. @qiyunzhu do you agree?